src/target/imxrt.c

```diff
@@ -154,14 +193,14 @@ static void imxrt_spi_read(target_s *target, uint16_t command, target_addr_t add
 static void imxrt_spi_write(
 	target_s *target, uint16_t command, target_addr_t address, const void *buffer, size_t length);
 static void imxrt_spi_run_command(target_s *target, uint16_t command, target_addr_t address);
+static void imxrt_id_chip(target_s *target, imxrt_priv_s *priv);
 
 bool imxrt_probe(target_s *const target)
 {
 	/* If the part number fails to match, instantly return. */
-	if (target->part_id != 0x88cU)
+	if (target->part_id != 0x88cU && target->part_id != 0x88c6U) {
```

Comment:
Please remove the addition of the braces. They're unnecessary.

Reply:
Would this clang-format setting work to automatically get braces correct? https://clang.llvm.org/docs/ClangFormatStyleOptions.html#removebracesllvm This would be nice to automate.

Reply:
Unfortunately, and because of that big fat warning listed, it wouldn't quite do the right thing and, as they note, would lead to incorrect code in some instances. That is why we elected to leave it and the next one about excess parens off